src/otx/cli/cli.py

```diff
@@ -422,6 +422,14 @@ def instantiate_model(self, model_config: Namespace) -> tuple:
                 warn(warning_msg, stacklevel=0)
                 model_config.init_args.num_classes = num_classes
 
+            # Hlabel classification
+            from otx.core.data.dataset.classification import HLabelInfo
```

Comment:
Can we move the import higher so as not to mix with the code? Either in this function or at the beginning of the file.

Reply:
Unfortunately, the CLI doesn't assume torch or lightning is installed by default, so this is a good place to be.